preprocessors/openstreetmap/osm_service.py

```diff
@@ -941,3 +946,62 @@ def get_coordinates(content):
     """
     if 'coordinates' in content.keys():
         return content['coordinates']
+
+    google_api_key = os.environ["GOOGLE_PLACES_KEY"]
+
+    # Query google places API to find latlong
+    request = f"https://maps.googleapis.com/maps/api/place/details/json?\
+            place_id={content['placeID']}&\
+            key={google_api_key}"
+
+    request = request.replace(" ", "")
```

Comment:
Where are spaces coming from in here?

Reply:
The next line character ```/``` is used to make the string length, for the API request, comply with the PEP8 standard. It however introduces spaces to the API request as shown below. The spaces are therefore removed by the second command line. 


``` "https://maps.googleapis.com/maps/api/place/details/json?            place_id=ChIJ__F4RUYayUwRjMCZ50oRRLA&            key=API_KEY"
```